swim/join_test.go

```diff
@@ -212,3 +212,15 @@ func (s *JoinSenderTestSuite) TestCustomDelayer() {
 func TestJoinSenderTestSuite(t *testing.T) {
 	suite.Run(t, new(JoinSenderTestSuite))
 }
+
+func (s *JoinSenderTestSuite) TestDontDisseminateJoinList() {
+	tnodes := genChannelNodes(s.T(), 5)
+	bootstrapNodes(s.T(), tnodes...)
+
+	for i, tnode := range tnodes {
+		s.Len(tnodes[i].node.memberlist.GetMembers(), i+1, "expected that next node joined all previous ones")
+		s.Len(tnodes[i].node.disseminator.changes, 1, "expected to only disseminate yourself")
```

Comment:
can you make this Disseminator API calls? e.g. (`ChangeByAddress()` and `ChangeCount()`)